__tests__/Unit/Components/Tasks/TasksContent.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { Provider } from 'react-redux';
+import { renderWithRouter } from '@/test_utils/createMockRouter';
+import { setupServer } from 'msw/node';
+import handlers from '../../../../__mocks__/handlers';
+import { noTasksFoundHandler } from '../../../../__mocks__/handlers/tasks.handler';
+import { store } from '@/app/store';
+import { TasksContent } from '@/components/tasks/TasksContent';
+
+const server = setupServer(...handlers);
+
+beforeAll(() => {
+    server.listen({ onUnhandledRequest: 'error' });
+});
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+describe('TasksContent', () => {
+    const loadingText = 'Loading...';
+    const NO_TASKS_FOUND_MESSAGE = 'No Tasks Found';
+
+    it('should render loading state', async () => {
+        const { getByText } = renderWithRouter(
+            <Provider store={store()}>
+                <TasksContent />
+            </Provider>
+        );
+
+        const loadingElement = getByText(loadingText);
+        expect(loadingElement).toBeInTheDocument();
+    });
+
+    it('should render tasks list', async () => {
+        const { findByText } = renderWithRouter(
+            <Provider store={store()}>
+                <TasksContent />
+            </Provider>
+        );
+        const tabNameElement = await findByText('MERGED');
+        expect(tabNameElement).toBeInTheDocument();
+        const taskTitleElement = await findByText('test for dashboard');
+        expect(taskTitleElement).toBeInTheDocument();
+        const taskStatusElement = await findByText('ASSIGNED');
+        expect(taskStatusElement).toBeInTheDocument();
```

Comment:
A better way is to use one of the tasks from the mock and then test if the task content is rendered

Reply:
How can I do it?

Reply:
Please ignore this comment. I believe that the tasks handler does not take mock data from fixtures/ __mock__/db